src/AbstractFunctionReferenceFixer.php

```diff
@@ -67,7 +67,7 @@ protected function find($functionNameToSearch, Tokens $tokens, $start = 0, $end
         $matches = $tokens->findSequence($candidateSequence, $start, $end, false);
         if (null === $matches) {
             // not found, simply return without further attempts
-            return;
+            return null;
```

Comment:
If I would compare it to perl - PHP7 void make a function work in empty context, so no return data is passed - null or not. you cannot grab the outcome of that function.
I agree that from functional point of view there is no difference, still, I wanted to solve issue raised by SCA.